routes/talks.js

```diff
@@ -53,12 +53,19 @@ function talks(req, res) {
             description
           } = extractTalk(body)
 
+          let isLightningTalk = false
+          for (const obj of labels) {
+            if (obj.name === 'Lightning Talk') {
+              isLightningTalk = true
+            }
+          }
```

Comment:
This or as oneliner `const isLightningTalk = labels.includes('Lightning Talk')`? 😄 

Also should `Lightning Talk` be filtered out of `labels`?

Reply:
`labels` is an array containing objects, so unfortunately `Array.prototype.includes()` won't work here I think. The idea with filtering it out makes sense tho, on it